apps/web/src/components/editor/index.tsx

```diff
@@ -59,7 +59,7 @@ export const Editor = memo(
       <div className="overflow-y-auto">
         <EditorRoot>
           <EditorContent
-            className="h-full w-full p-1"
+            className="prose-a:cursor-pointer h-full w-full p-1"
```

Comment:
don't put the class here
put it while other `prose-` classes are in `editorProps`